ginga/meta.yaml

```diff
@@ -1,6 +1,6 @@
 {% set name = 'ginga' %}
 {% set version = '2.5.20160627100500' %}
```

Comment:
Can we change the timestamp ourselves even if it is not the same as version set by Ginga itself?